src/Prometheus/PushGateway.php

```diff
@@ -88,9 +96,9 @@ private function doRequest(CollectorRegistry $collectorRegistry, string $job, ar
             $renderer = new RenderTextFormat();
             $requestOptions['body'] = $renderer->render($collectorRegistry->getMetricFamilySamples());
         }
-        $response = $client->request($method, $url, $requestOptions);
+        $response = $this->client->request($method, $url, $requestOptions);
         $statusCode = $response->getStatusCode();
-        if ($statusCode != 202) {
+        if ($statusCode !== 200 && $statusCode !== 202) {
```

Comment:
Tip: `if (!in_array($statusCode, [200, 202]))`
Why? If new status will be used, we need only add to the array.

Reply:
thanks for that advice, done!